docs/latest/tutorial/tutorial-scaffolding.md

```diff
@@ -0,0 +1,354 @@
+
+```json {8-10} title='package.json'
+{
+  "name": "my-electron-app",
+  "version": "1.0.0",
+  "description": "Hello World!",
+  "main": "main.js",
+  "author": "Jane Doe",
+  "license": "MIT",
+  "scripts": {
+    "start": "electron ."
+  }
+}
+```
+
+This `start` command will let you open your app in development mode.
+
+```sh npm2yarn
+npm start
+```
+
+:::tip
+This script tells Electron to run on your project's root folder. At this stage,
+your app will immediately throw an error telling you
+
+> "Error launching app"
+
+This is expected as we have not writen any code yet!
+:::
+
+## Adding the entry point
+
+The entry point of any Electron application is its `main` script. This script controls the
+**main process**, which runs in a full Node.js environment and is responsible for
+controlling your app's lifecycle, displaying native interfaces, performing privileged
+operations, and managing renderer processes (more on that later).
+
+During execution, Electron will look for this script in the [`main`][package-json-main]
+field of the app's `package.json` config, which you should have configured in the previous
+step.
+
+To initialize the `main` script, create a file named `main.js` in the root folder
+of your project with the following contents and run `npm start` again:
+
+```js title='main.js'
+console.log(`Hello from Electron 👋`);
+```
+
+In your console you should read `Hello from Electron 👋`.
+
+Congratulations, you have executed your first Electron application!
+
+:::tip
+If you haven't yet, please read [Electron's process model][process-model] to better understand
+how all the pieces work together.
+:::
+
+## Adding some UI
+
+Before we can create a window for our application, we need to create the content that
+will be loaded into it. In Electron, each window displays web contents that can be loaded
+from either from a local HTML file or a remote URL.
+
+For this tutorial, you will be doing the former. Create an `index.html` file in the root
+folder of your project:
+
+```html title='index.html'
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <!-- https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP -->
+    <meta
+      http-equiv="Content-Security-Policy"
+      content="default-src 'self'; script-src 'self'"
+    />
+    <meta
+      http-equiv="X-Content-Security-Policy"
+      content="default-src 'self'; script-src 'self'"
+    />
+    <title>Hello from Electron renderer!</title>
+  </head>
+  <body>
+    <h1>Hello from Electron renderer!</h1>
+    <p>👋</p>
+  </body>
+</html>
+```
+
+Now that you have a web page, load it into an application window. To do so, paste
+the following code in your `main.js` file so we can disect it line by line:
+
+```js title='main.js'
+const { app, BrowserWindow } = require('electron')
+
+const createWindow = () => {
+  const win = new BrowserWindow({
+    width: 800,
+    height: 600,
+    webPreferences: {
+      sandbox: true,
```

Comment:
I think we should align with `electron-quick-start` for props here. Not sure if we should be including `sandbox` without greater alignment around `sandbox: true` across the docs, or at least more explanation on what the prop does here and linking to the Sandboxing doc.

Also unrelated but there's a trailing comma here but none in the next instance of this snippet.

Reply:
weird that the comma isn't failing in the pr to e/e.

I'd rather use `sandbox` in the tutorial. That's what we want people to use and the default in this cases should be `true` soon-ish when not specified.

Reply:
```suggestion
      sandbox: true
```